views/delivered.ejs

```diff
@@ -179,17 +195,13 @@
                                       valign="top"
                                       bgcolor="">
                                       <div style="text-align: center;"><span style="color:#333333;">
-                                        <p><%= i18n.info %> <a href="<%= store.homepage %>/app/#/account/orders"><%= i18n.my_acc %></a> <%= i18n.contact_us %> </p>
+                                        <p><%= i18n.info %> <a href="<%= store.homepage %>"><%= i18n.my_acc %></a> <%= i18n.contact_us %> </p>
```

Comment:
you're linking to homepage but the link text is "my account"?

```suggestion
                                        <p><%= i18n.info %> <a href="<%= store.domain %>/app/#/account/"><%= i18n.my_acc %></a> <%= i18n.contact_us %> </p>
```

Reply:
It is wrong in this file, but in the others it's ok